docs/src/lib/shadcn/components/ui/chart/chart-tooltip.svelte

```
@@ -1,159 +1,159 @@
-                {
-                    value: unknown
-                    name: string
-                    item: TooltipPayload
-                    index: number
-                    payload: TooltipPayload[]
-                }
-            ]
-        >
-    } = $props()
+	let {
+		ref = $bindable(null),
```

ESLint

[error] This assigned value is not used in subsequent statements.

(no-useless-assignment)